src/main/webapp/app/course/competencies/competency-management/competency-management.component.ts

```diff
@@ -62,6 +63,7 @@ export class CompetencyManagementComponent implements OnInit, OnDestroy {
     private readonly activatedRoute: ActivatedRoute = inject(ActivatedRoute);
     private readonly competencyService: CompetencyService = inject(CompetencyService);
     private readonly prerequisiteService: PrerequisiteService = inject(PrerequisiteService);
+    private readonly courseCompetencyService: CourseCompetencyService = inject(CourseCompetencyService);
```

Comment:
Why do we need all 3 services? Shouln't be the CourseCompetencyService enough for this use case?

Reply:
I still think we just need the CourseCompetencyService here as we also get the prerequisites and competency from there (and we need only one server request for that)

Reply:
Already replaced